contrib/aleph-traefik-minio-keycloak/docker-compose.yml

```diff
@@ -65,7 +65,7 @@ services:
       - "traefik.enable=false"
 
   worker:
-    image: alephdata/aleph:${ALEPH_TAG:-3.12.2}
+    image: ghcr.io/alephdata/aleph:${ALEPH_TAG:-3.12.2}
```

Comment:
Whilst we are here can we update the tags to be 3.14.0?

Reply:
Right! In that case I’d also add the file to the bump2version config, ok?

I was just thinking that maybe it’s a good idea for these example configuration to be actually a "frozen" snapshot, i.e. it worked with 3.12.2 but we don’t really maintain or test these -- so it might be better to not update it at all (which includes reverting the new repo, because 3.12.2 is still available on Docker Hub).